services/movie.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
+) -> List[Movie]:

```

Comment:
This i a wrong annotation, fix everywhere

Reply:
@ABaidin still not fixed
Also genres_ids and actors_ids are lists or None